src/main/features/core/controlBar.js

```diff
@@ -4,10 +4,19 @@ Emitter.on('window:minimize', (event, windowID) => {
 
 Emitter.on('window:maximize', (event, windowID) => {
   const window = WindowManager.getByInternalID(windowID);
-  if (window.isMaximized()) {
-    WindowManager.getByInternalID(windowID).unmaximize();
+
+  if (process.platform === 'darwin') {
+    if (window.isFullScreen()) {
```

Comment:
Could trim this down to one line with `window.setFullScreen(!window.isFullScreen())`